src/rotten-tomatoes/index.js

```diff
@@ -1,3 +1,111 @@
-function rottenTomatoes(grid) {}
+//helper function
+
+function findFresh(map) {
+    for (let row = 0; row < map.length; row++) {
+        for (let column = 0; column < map[row].length; column++) {
+            if (map[row][column] == 1) {
+                //check that there is a fresh tomato that can be rottened
+                //up
+                if (row - 1 >= 0) {
+                    if (map[row - 1][column] == 2) {
+                        return true;
+                    }
+                }
+                //down
+                if (row + 1 < map.length) {
+                    if (map[row + 1][column] == 2) {
+                        return true;
+                    }
+                }
+                //left
+                if (column - 1 >= 0) {
+                    if (map[row][column - 1] == 2) {
+                        return true;
+                    }
+                }
+                //right
+                if (column + 1 < map[row].length) {
+                    if (map[row][column + 1] == 2) {
+                        return true;
+                    }
+                }
+            }
+
+        }
+    }
+    return false;
+}
+
+function makeRotten(map, array) {
+    while (array.length != 0) {
+        //take first 2 indexes as x and y cordinates of the grid and make rotten
+        var row = array[0];
+        var column = array[1];
+        map[row][column] = 2;
+        //revove first 2 making the next 2 the cordinates
+        array.shift();
+        array.shift();
```

Comment:
Why do you need to shift twice? Someone might see this and assume it's an error.